src/components/v2/appDetails/AppDetails.component.tsx

```diff
@@ -174,7 +173,7 @@ const AppDetailsComponent = ({
                 />
                 {!appDetails.deploymentAppDeleteRequest && (
                     <EnvironmentStatusComponent
-                        appStreamData={streamData}
+                        resourcesSyncResult={appDetails.resourceTree.resourcesSyncResult}
```

Comment:
Is this expected that `resourcesSyncResult` will be always be in resourceTree

Reply:
Any specific reason to pass from here?

Reply:
Have done optional chaining for ResourceTree

Reply:
Removed the prop from here since its available from appDetails inside component